Uc2 (#52)
materials-marketplace/user-docs: docs/source/ucs/sa_weld_strength_tut.md

# Standalone app: Weld strength

## Description

This app calculates the strength of a weld based on user-provided parameters.

> The user select the desired weld type, and the relevant parameters such as weld throat thickness and effective length.

Author: ```suggestion
The user selects the desired weld type, and the relevant parameters such as weld throat thickness and effective length.
```

Depending on the selected weld type and load configuration, the function calculates and displays the corresponding stress values.

For butt welds, it considers various load scenarios such as tensile, shear, bending, and twisting.
For fillet welds, it calculates the strength based on different fillet weld configurations.

> ## How ro run the app

Author: ```suggestion
## How to run the app
```

### Inputs

> When you open the app you will be presented by the following screen

Author: ```suggestion
When you open the app you will be presented with the following screen
```

![ws_app|200x100](../_static/img/ucs/standalone_apps/weld_input_1.png)

You have to select the type of weld to calculate (butt weld or fillet weld), and you also have to provide the weld throat thickness and effective length of the weld.

![ws_app|200x100](../_static/img/ucs/standalone_apps/weld_input_2.png)

> Here we selct the butt weld, weld throat thickness of 2 mm and effective length of the weld of 2 mm.

Author: ```suggestion
Here we select the butt weld, weld throat thickness of 2 mm and effective length of the weld of 2 mm.
```

> Once the butt weld type is selected you hav to select a configuration, to hel you decide the app shows a image depicting the available possibilities.

Author: ```suggestion
Once the butt weld type is selected you have to select a configuration to help you decide the app shows a image depicting the available possibilities.
```

![ws_app|200x100](../_static/img/ucs/standalone_apps/weld_input_3.png)

> As a example let's select the twist case with a torque of 1000 N.

Author: ```suggestion
As an example let's select the twist case with a torque of 1000 N.
```

> A this point you only have to click on compute.

Author: ```suggestion
At this point you only have to click on compute.
```

### Results

After processing the input, the app will present you with the results. The output will be displayed as follows:

![ws_app|200x100](../_static/img/ucs/standalone_apps/weld_output_1.png)

> In this screen, you have a summary of your imputs. togheter witht the resulting stress.

Author: ```suggestion
In this screen, you have a summary of your inputs together with the resulting stress.
```
